src/sagemaker/jumpstart/artifacts.py

```diff
@@ -281,7 +281,10 @@ def _retrieve_script_uri(
     )
 
     if script_scope == JumpStartScriptScope.INFERENCE:
-        model_script_key = model_specs.hosting_script_key
+        model_script_key = (
+            getattr(model_specs, "training_prepacked_script_key") or model_specs.hosting_script_key
+        )
```

Comment:
question: shouldn't there be a way to access the uncombined model artifact?

Could you please align with @whittech1 on this?

Reply:
We decided that there is no harm in including the script with the model artifact, even if someone only wants the model artifact. Most of the combined tarball is the model artifact, so it doesn't make a huge difference in terms of storage

Reply:
The risk is that there are extra files left over from the combined model artifact in the `code` dir when a user tries to override the script. In the worst case, it could make the new combined script crash. Maybe we should sleep on this.